### ISSUE TITLE
topBar title component alignment ignored after pushing and popping off of stack

### ISSUE THREAD

**Issue by donovanhiland** [wix/react-native-navigation#4794]
### Issue Description

I'm rendering a custom topBar title component with these options:
```js
  static get options() {
    return {
      topBar: {
        drawBehind: false,
        title: {
          component: {
            name: 'navigation.Logo',
            alignment: 'center',
          },
        },
      },
    }
  }
```
Which is correctly centering the custom component. However, when I push a view onto the stack and navigate back the custom component is no longer centered.

### Steps to Reproduce / Code Snippets / Screenshots

I can repro this in the playground app. Start playground app, tap Options on the bottom, tap Set React Title View, tap Push, tap Pop or Pop to Root. The custom component is no longer centered.

![image](https://user-images.githubusercontent.com/17991396/53456504-eeeb3300-39eb-11e9-8033-01f7cb50cea9.png)
![image](https://user-images.githubusercontent.com/17991396/53456521-00ccd600-39ec-11e9-9c07-36b28705c5c0.png)

---
### Environment
* React Native Navigation version: 2.11.0
* React Native version: 0.58.5
* Platform(s) (iOS, Android, or both?): iOS, haven't tested android
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator

^ These are the versions I'm seeing this issue with. It's also happening with the playground versions.

**Comment by brenwell** [comment]
Having the same issue. The following is not working either

```typescript
    Navigation.setDefaultOptions({
      topBar: {
        background: {
          color: backgroundColor,
        },
        title: {
          alignment: `center`
        },
        backButton: {
          color: titleStyle.color,
        },

      },

    })
```

**Comment by gustavomts** [comment]
Same thing for me.
`Navigation.setDefaultOptions({
      statusBar: {
        visible: true,
        style: 'light'
      },
      topBar: {
        visible: true,
        title: {
          component: { name: LOGO_ICON },
          alignment: 'center'
        },
        background: { color: '#1d1d1d' },
        backButton: { visible: false, },
        noBorder: true,
      },
      layout: {
        orientation: ['portrait'],
      },
    });`

**Comment by yfuks** [comment]
I commented the following line 
https://github.com/wix/react-native-navigation/blob/c6b6f731f4f0288da9c01dec3088dc4b903f8baf/lib/ios/RNNViewControllerPresenter.m#L183

And it fix the issue in my case

**Comment by brenwell** [comment]
That also worked for me

**Comment by eyes2design** [comment]
IOS fix works for me as well.
Using
React Native: 0.58.4

**Comment by ddinggu** [comment]
RN: 0.58.4
RNN V2: 2.11.0 

also works for me

**Comment by eyes2design** [comment]
I hope they will get this into a update patch soon or something like it.

**Comment by hahtml** [comment]
This fix works for me.

**Comment by brenwell** [comment]
I am doing this temporarily in `postInstall`. For anyone looking for a hack ;)

```shell
find . -name 'RNNViewControllerPresenter.m' -exec sed -i -e '/\[\_customTitleView setFrame\:frame];/s/^/\/\//g' {} \;
```

**Comment by Pungsnigel** [comment]
Fix worked for me as well.

**Comment by yfuks** [comment]
Looks like it was fixed with this commit: https://github.com/wix/react-native-navigation/commit/ac60d2fe6ad036528c31954a2997109b06f0c947#diff-8e71e0bc0406a74d2ee34eb8926fc021

So it should be ok with **2.17.0** can someone confirm?
I can't try right now

**Comment by hahtml** [comment]
@yfuks Yeap, it was fixed in **2.17.0**
This fix should be mentioned in Changelog as well
